go/vt/vttablet/tabletmanager/rpc_external_reparent.go

```diff
@@ -160,9 +161,14 @@ func (agent *ActionAgent) finalizeTabletExternallyReparented(ctx context.Context
 		}
 	}()
 
+	tablet := agent.Tablet()
+	tabletMap, err := agent.TopoServer.GetTabletMapForShard(ctx, tablet.Keyspace, tablet.Shard)
+	// make the channel buffer big enough that it doesn't block senders
+	tablets := make(chan topodatapb.Tablet, len(tabletMap))
 	if !topoproto.TabletAliasIsZero(oldMasterAlias) {
 		wg.Add(1)
 		go func() {
+			log.Infof("finalizeTabletExternallyReparented: updating tablet record for old master: %v", oldMasterAlias)
```

Comment:
By convention, `defer wg.Done()` should remain the first line so it's immediately clear that it always gets called.